openapi/resource-server.yaml

```diff
@@ -1193,6 +1193,10 @@ components:
           description: "The total amount that should be deducted from the sender's account when the corresponding outgoing payment has been paid. "
         method:
           $ref: '#/components/schemas/payment-method'
+        estimatedExchangeRate:
+          type: number 
+          description: 'Estimated probed exchange rate over the path (ILP specific).'
+          readOnly: true
```

Comment:
How will the `estimatedExchangeRate` be returned in this case? IIRC it should take into consideration exchange rate and fees (if they exist). Will this be computed in the RS and returned as a ready to use value for the consumer?